sdk/identity/identity/test/manual-integration/AzureFunctions/INSTRUCTIONS.md

```diff
@@ -0,0 +1,50 @@
+# Testing Identity in Azure Functions
+
+## Configuration
+
+Before you begin, you must have the following:
+
+An Azure account with an active subscription.
+
+The Azure Functions Core Tools version 3.x.
+
+One of the following tools for creating Azure resources:
+
+Azure CLI version 2.4 or later.
+
+Azure PowerShell version 5.0 or later.
+
+Node.js, Active LTS and Maintenance LTS versions (8.11.1 and 10.14.1 recommended).
+
+- Prerequisites and Configuration of local env same as this - https://docs.microsoft.com/azure/azure-functions/create-first-function-cli-typescript?tabs=azure-cli%2Cbrowser#configure-your-local-environment
+
+## Prerequisites
+
+In a terminal or command window, run func --version to check that the Azure Functions Core Tools are version 3.x.
+
+Run az --version to check that the Azure CLI version is 2.4 or later.
+
+Run az login to sign in to Azure and verify an active subscription.
+
+- An Azure Key Vault.
+
+### Install Azure Functions
+
+`cd IdentityTest`
+`npm install`
+
+## Give the Azure Functions access to the key vault
```

Comment:
This could just be "Deploy the required resources and set up permissions" and then you can combine the steps above in `Install Azure Functions` with this. 

Basically we can have:

- prerequisites (things you must already have installed)
- setup / config - npm install the things, create resource group, deploy arm template, set up permissions, etc.
- Deploy and run - deploy the function itself and test it out